pyproject.toml

```diff
@@ -65,6 +65,10 @@ bandit = "^1.7.4"
 "Bug Tracker" = "https://github.com/frenck/python-adguardhome/issues"
 Changelog = "https://github.com/frenck/python-adguardhome/releases"
 
+[tool.poetry.group.dev.dependencies]
+pytest-xdist = "<2.0"
+icecream = "^2.1.3"
```

Comment:
I added this for pretty printing in debugging; suuuuper useful when debugging async calls!

Reply:
printing != debugging. 
Use the debugger if you need debugging. Please remove it, there is no need to add another dependency.